src/Shared/PackageManagers/BaseProjectManager.cs

```diff
@@ -323,7 +324,7 @@ public async Task<string> ExtractArchive(string directoryName, byte[] bytes, boo
                 Parallel = true
                 // MaxExtractedBytes = 1000 * 1000 * 10;  // 10 MB maximum package size
             };
-            ExtractionStatusCode result = await extractor.ExtractToDirectoryAsync(TopLevelExtractionDirectory, dirBuilder.ToString(), new MemoryStream(bytes), extractorOptions);
+            ExtractionStatusCode result = await extractor.ExtractToDirectoryAsync(topLevelDirectory, dirBuilder.ToString(), new MemoryStream(bytes), extractorOptions);
```

Comment:
You end up turning the bytes back into a Stream before calling extract anyway. Recommend you just pass the Stream.

Reply:
Switched to passing the stream around instead.